nt-be/migrations/20251228000002_add_block_time.sql

```diff
@@ -0,0 +1,17 @@
+-- Add block_time as timestamptz for API responses
+ALTER TABLE balance_changes 
+ADD COLUMN block_time TIMESTAMPTZ;
+
+-- Populate block_time from block_timestamp (nanoseconds since Unix epoch)
+-- NEAR timestamps are in nanoseconds, convert to seconds for timestamp
+UPDATE balance_changes 
```

Comment:
a bit confusing to have them both..,.

Reply:
yes I started with the nanoseconds timestamp, the raw value from the block header, but then realised that it is more useful to have something that is more readable in the database. Let me keep both for now, and rather consider removing the nanosecond timestamp later.